docs/writing-stories/args.md

```diff
@@ -88,7 +88,8 @@ Args are useful when writing stories for composite components that are assembled
   paths={[
     'react/page-story.js.mdx',
     'react/page-story.ts.mdx',
-    'angular/page-story.ts.mdx'
+    'angular/page-story.ts.mdx',
+    'vue/page-story.ts.mdx'
```

Comment:
```suggestion
    'vue/page-story.js.mdx'
```

Reply:
🤦 Sorry about that, is there any way i can manually check that it is being rendered correctly?

Reply:
These pages get slurped in by https://github.com/storybookjs/frontpage, so to see it all working you need to set that up as described in the README.

Reply:
I rebased to apply the change to leave a cleaner git history. I'm not able to render the docs following the link that you shared @shilman. Gatsby is not able to get all the entries for me in master :(